### PR TITLE
Fillmore haight

### PR DESCRIPTION
**Pull request by mojavedejavu**
lease review this, @casey-collab

### CODE REVIEW

File: english/sanfran/bikeride/ginger-peachy-keen/drink-more-teas/explosive-euphoria.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The world is turning n I'm exploding...Hush-hush I think I've had enough tea for now.
```
Comment (casey-collab) [inline english/sanfran/bikeride/ginger-peachy-keen/drink-more-teas/explosive-euphoria.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/sanfran/bikeride/ginger-peachy-keen/drink-more-teas/morning-jade.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Hmmmmmmm YUM!!!
```
Comment (casey-collab) [inline english/sanfran/bikeride/ginger-peachy-keen/drink-more-teas/morning-jade.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mojavedejavu** [comment]
lease review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**